src/Common/src/System/Xml/Base64Encoder.cs

```diff
@@ -5,9 +5,9 @@ namespace System.Xml
 {
     internal abstract partial class Base64Encoder
     {
-        byte[] leftOverBytes;
-        int leftOverBytesCount;
-        char[] charsLine;
+        private byte[] leftOverBytes;
```

Comment:
This applies to all of the fields here, BTW. Some of the classes here do seem to follow the underscore convention so it's a little inconsistent.